src/Type/UnionIterableType.php

```diff
@@ -2,14 +2,18 @@
 
 namespace PHPStan\Type;
 
-class UnionIterableType implements IterableType, UnionType
+class UnionIterableType implements UnionType
 {
 
 	use IterableTypeTrait;
 
 	/** @var \PHPStan\Type\Type[] */
 	private $types;
 
+	/**
+	 * @param Type   $itemType
```

Comment:
ok

---

<small>*(I prefer aligning phpDoc as it greatly increases readability and good diff tools such as [reviewable](https://reviewable.io) can show the diff clearly. I wish all diff tools were that smart)*</small>